NEWS

```diff
@@ -9,6 +9,14 @@ Next
 Changes
 -------
 
+* ``AsynchronousDeferredRunTest`` now has ``suppress_twisted_logging`` and
+  ``store_twisted_logs`` parameters that can be used to override the default
+  logging behaviour.  (Jonathan Lange, #942785)
+
+* New fixture ``CaptureTwistedLogs`` that can be used with
+  ``AsynchronousDeferredRunTest`` to attach a detail containing everything
```

Comment:
Its a twisted specifical log capturerer, but not specific to testtools; thats true. OTOH testtools is currently the only twisted and details supporting runner; I think being overly general will make the changelog harder to understand.

Which reminds me: docs please; shouldn't this be in the for-test-authors docs?